cmd/sink-worker/main.go

```diff
@@ -162,41 +166,46 @@ func main() {
 		logger.Error("failed to initialize sink worker", "error", err)
 		os.Exit(1)
 	}
-
-	var group run.Group
+	defer func() {
+		_ = sink.Close()
```

Comment:
shouldn't we log error?

Reply:
Actually we do within `sink.Close()` function. I'll push a followup PR which will remove `error` retrun for `sink.Close()` as does not return error anyway.